modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -75,11 +75,52 @@ function inope_policy_review_permission() {
 }
 
 /**
+ * Implements hook_form_alter().
+ */
+function inope_policy_review_form_alter(&$form, &$form_state, $form_id) {
```

Comment:
Why are you doing this in an alter hook and not in the page callback or the form callback?